src/psyclone/transformations.py

```diff
@@ -325,6 +325,9 @@ class OMPTaskloopTrans(ParallelLoopTrans):
     :type grainsize: int or None
     :param num_tasks: the num_tasks to use for this transformation.
     :type num_tasks: int or None
+    :param nogroup: Whether of not to use a nogroup clause for this
+                    transformation. Default is False.
+    :type nogroup: bool
```

Comment:
As this is an intrinsic type you can specify it in-line in the param description:

    :param bool nogroup: .....

Reply:
Done